Dockerfile

```diff
@@ -32,4 +32,4 @@ RUN ln -s /usr/bin/openshift-builder /usr/bin/openshift-sti-build && \
 LABEL io.k8s.display-name="OpenShift Builder" \
       io.k8s.description="This is a component of OpenShift and is responsible for executing image builds." \
       io.openshift.tags="openshift,builder"
-ENTRYPOINT [ "/usr/bin/entrypoint.sh" ]
+ENTRYPOINT [ "/usr/bin/entrypoint.sh" ]
```

Comment:
Drop this - I don't think it is needed, and it will create a merge conflict with #414